src/cmd/cmd_sym.ml

```diff
@@ -71,10 +72,23 @@ let print_bug ~model_format ~model_out_file ~id ~no_value ~no_stop_at_failure
               }
             ]
           in
-          model.children @ lbls @ bcrumbs
-        else model.children @ lbls
+          { model with children = model.children @ bcrumbs }
+        else model
       in
-      Scfg.Pp.directive fmt { model with children }
+      let model =
+        match entry_point with
+        | Some entry_point ->
+          let ep =
+            [ { Scfg.Types.name = "entry_point"
+              ; params = [ entry_point ]
+              ; children = []
+              }
+            ]
+          in
+          { model with children = model.children @ ep }
+        | None -> model
+      in
```

Comment:
Do you think we could put the entry point first in the output model (followed by the symbols)? I find it easier to read this way